sbin/create-swap.sh

```diff
@@ -5,8 +5,23 @@ set -x
 
 # Calculate the ideal swap size.
 # There should be 32 GB of temporary file system size (RAM and swap combined).
-# First, we fnd the RAM size in GB by converting from KB.
+# First, we find the RAM size in GB by converting from KB.
 ram_gb=$(expr $(cat /proc/meminfo | grep MemTotal | awk {'print $2'}) / 1000000)
+root_size=$(df -h | grep -P '\/$' | awk '{print $4}')
```

Comment:
In ostree based systems `df` output shows the root as mounted on `/sysroot`, there is actually no entry shown to be mounted on `/`. I verified this both on my Silverblue install and on a PlaytronOS install.

This script is run post-install on first boot inside PlaytronOS, so we need to test in that environment.